lib/function.g

```diff
@@ -159,6 +159,31 @@ BIND_GLOBAL( "TYPE_OPERATION",
 ##
 DeclareOperationKernel("NameFunction", [IS_OBJECT], NAME_FUNC);
 
+#############################################################################
+##
+#O  NameFunction( <func> )  . . . . . . . . . . . . . . .  name of a function
+##
+##  <#GAPDoc Label="HasNameFunction">
+##  <ManSection>
+##  <Oper Name="HasNameFunction" Arg='func'/>
+##
+##  <Description>
+##  returns if a function has an assigned name.
+##  <P/>
+##  <Example><![CDATA[
+##  gap> NameFunction(SylowSubgroup);
+##  "SylowSubgroup"
+##  gap> HasNameFunction(SylowSubgroup);
+##  true
+##  gap> HasNameFunction(x->x);
```

Comment:
Maybe I miss something, but how does this work?

Reply:
How does what work?

Reply:
That `x->x` has a name? It's not set in this example. Or it is autogenerated? What it will be in this case?

Reply:
Ah, there was a typo in the test, now fixed :)